samples/Mvc.ActionConstraintSample.Web/web.Debug.config

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Suggest adding `web.Debug.config` and `web.Release.config` to `.gitignore` rather than checking these in.